### PR TITLE
build app.jar and publish to maven central

### PR DESCRIPTION
**Pull request by anupkumangodan**
Resolves #

## Description
Build yavin app.jar and publish to maven central.

## Proposed Changes

- add maven publication steps in gradle
- update README
- decrypt script to help in gpg signing
- add gradle.properties to allow passing variables required for publish steps
- add encrypted gpg ring files
- requires - https://github.com/yavin-dev/app/pull/26 (screwdriver yaml changes)

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: ui/package.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
   },
   "main": "index.js",
   "scripts": {
-    "build": "ember build",
+    "build": "DEBUG=ember-cli:* ember build",
```
Comment (jkusa) [inline ui/package.json:16]:
```suggestion
    "build": "ember build",
```
Can this be reverted?

File: ws/build.gradle.kts [hunk 2]
```diff
@@ -64,12 +67,78 @@ tasks.withType<Test> {
...
+                url.set("http://yavin.dev")
+                licenses {
+                    license {
+                        name.set("MIT")
+                    }
+                }
+                developers {
+                    developer {
+                        id.set("yavin-dev")
+                        name.set("Yavin Development Team @VerizonMedia")
```
Comment (anupkumangodan) [inline ws/build.gradle.kts:112]: add env vars for the name, email, scm info.

**Review by anupkumangodan — COMMENTED** [review]
few more

**Comment by anupkumangodan** [comment]
pending

1. move pub ring enc files to screwdriver branch
2. move properties from gradle.properties file into build file

File: ws/build.gradle.kts [hunk 3]
```diff
@@ -64,12 +67,85 @@ tasks.withType<Test> {
...
     dependsOn("bootJar")
     commandLine = listOf("java", "-jar", "${project.buildDir}/libs/${jarName}.jar")
 }
 
+
 tasks.bootJar {
     archiveBaseName.set(jarName)
 }
+
+// PUblishing to Maven central
```
Comment (jkusa) [inline ws/build.gradle.kts:81]:
```suggestion
// Publishing to Maven central
```

File: ws/build.gradle.kts [hunk 4]
```diff
@@ -64,12 +67,85 @@ tasks.withType<Test> {
...
+group = "dev.yavin"
+version = "0.1"
```
Comment (jkusa) [inline ws/build.gradle.kts:83]: technically our version is `1.0.0-beta.4 `

**Review by jkusa — APPROVED** [review]

**Pull request merged**